README.md

```diff
@@ -61,13 +63,23 @@ That's it.
 
 ### Installation via Composer
 
-If you like Composer:
+If you like Composer,Add phing in composer.json: 
+
+~~~
+"require": {
+    "php": ">=5.4.0",
+    "nikic/php-parser": "^2.1|^3.0|^4.2",
+    "phing/phing": "2.*",
+    "phpunit/phpunit": "4.8.16"     
+}
+~~~
```

Comment:
@biechao Remove this change, please. Phing is optional. Not every user needs it.